frontend/src/Constants.js

```diff
@@ -147,7 +147,7 @@ export const OTHER_ENTITY_TYPES = [
 // - src/constants.js
 export const TOPICS = [
   'Behavioral / Mental Health / Trauma',
-  'Child Assessment, Development, Screening',
```

Comment:
these will also need to be updated in ```src/constants.js```(I don't see that change in the other PR either)

I don't want to expand the scope of this PR, but we have a topics endpoint now - perhaps we can write a Jira to switch out of these hardcoded thingamabobs and use those where appropriate.

Reply:
thanks - somehow it looks like I also missed a whole bunch of other references to old topic names that I think should be updated, which I've done here: 6fbd748f34a5ec28e1eb7eb84262cbf97d7d0e0c

hopefully I caught them all in that commit ^^ tests appear to be passing, so hopefully that means 👍 

> perhaps we can write a Jira to switch out of these hardcoded thingamabobs and use the endpoint/info from DB

:100: agreed, good idea